docs/Compose.md

```diff
@@ -0,0 +1,181 @@
+For example, the [OSV](https://ossf.github.io/osv-schema/) certifier will watch
+GUAC for new packages, then try to discover OSV vulnerabilities for those
+packages. If any vulnerabilities are found, the certifier will attach a
+"CertifyVuln" node to the package that signifies that the package is connected
+to the OSV vulnerability.
+
+## Prerequisites
+
+- Docker
+- Docker Compose
+- Git
+- Go (optional)
+- Make (optional)
+
+## Clone GUAC
+
+If you haven't already, clone GUAC to a local directory:
+
+```bash
+git clone https://github.com/guacsec/guac.git
+```
+
+Optional: Also clone GUAC data, this is used as test data.
+
+```bash
+git clone https://github.com/guacsec/guac-data.git
+```
+
+The rest of the demo will assume you are in the GUAC directory
+
+```bash
+cd guac
+```
+
+## Build the containers
+
+The `Makefile` contains the `docker build` commands:
+
+```bash
+make container
+```
+
+> Note: you may also run the `docker build` command directly without `make`.
+
+## Start GUAC
+
+In another terminal, start up GUAC.
+
+```bash
+docker-compose up
+```
+
+The full GUAC deployment is now running. The GraphQL server is listening on port
+`8080`. You may visit [http://localhost:8080](http://localhost:8080) to see the
+GraphQL playground. GraphQL queries are served at the `/query` endpoint.
+
+Nats is listening on port `4222`, this is where any collectors that you run will
+need to connect to push any files they find. The GUAC `collector` command
+defaults to `nats://127.0.0.1:4222` for the Nats address, so this will work
+automatically.
+
+## Start Ingesting Data
+
+Now you can run any GUAC collector and push data to the running ingestion
+pipeline. For example we can ingest the sample `guac-data` data. However, you
+may ingest what you wish to here instead.
+
+```bash
+pushd ../guac-data/docs
+docker run --rm -v $PWD:/data --network guac_default local-organic-guac:latest /opt/guac/collector files /data --natsaddr nats:4222
+popd
+```
+
+Notice how the collectors run and complete quickly. If you have run the
+`guacone files` command in a previous demo, you know that that is a slower
+process. In this case the collector completes quickly because the parsing,
+computation, and ingesting is happening within the ingestion pipeline services
+that are running.
+
+Switch back to the compose window and you can see the ingestor logging the
+activity. Shortly after that you will see that the OSV certifier recognized the
+new packages and is looking up vulnerability information for them.
+
+This command uses the `collector` GUAC binary from the container we built
+earlier. Because containers don't have access to localhost by default, we
+connect to the compose network and use the `nats` container name to connect to
+it.
+
+Alternatively, you may build the GUAC binaries for your local machine and run
+them natively.
+
+```bash
+make build
+./bin/collector files ../guac-data/docs
+```
+
+or
+
+```bash
+go run ./cmd/collector files ../guac-data/docs
+```
+
```

Comment:
would be helpful to add a step here to show that the graphql api is queryable with the data. have some UI screenshots of the GraphQL interface.. and then followed by some placeholder for when we add the UI exploration options (which we will probably link out to another document).

Reply:
how about a HTTP query to the GQL endpoint just to show that it is working

Reply:
done